app/src/main/java/com/dessalines/thumbkey/MainActivity.kt

```diff
@@ -68,51 +70,51 @@ class MainActivity : AppCompatActivity() {
             ThumbkeyTheme(
                 settings = settings,
             ) {
-                if (appSettingsViewModel.appSettings.isInitialized) {
```

Comment:
This was causing the error, and was the wrong way to go about making sure settings are initialized.